compiler/verification/globals.ml

```diff
@@ -0,0 +1,28 @@
+(* This file is part of the Catala compiler, a specification language for tax
+   and social benefits computation rules. Copyright (C) 2022 Inria, contributor:
+   Aymeric Fromherz <aymeric.fromherz@inria.fr>, Denis Merigoux
+   <denis.merigoux@inria.fr>
+
+   Licensed under the Apache License, Version 2.0 (the "License"); you may not
+   use this file except in compliance with the License. You may obtain a copy of
+   the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+   License for the specific language governing permissions and limitations under
+   the License. *)
+
+let optimize_ref = ref false
+let disable_counterexamples_ref = ref false
+
+(** This sub-lib relies on global refs in many places. This should be cleaned
```

Comment:
Or this hack here?

Reply:
ah yes this needs to be cleaned up but I was a bit afraid to touch the verification code, which is why I left it like that but clearly delimited. As proposed it is on par with the previous behaviours.

Reply:
Ok then we can leave it as is for the moment :)